### PR TITLE
feat:codecept-playwright-sdk-launch

### PR DESCRIPTION
**Pull request by shayan-bstack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
-# codecept-js-playwright-browserstack
-Sample code for running sessions with Codecept Playwright on Browserstack
+# CodeceptJS-Playwright-Browserstack
+
+# Integrating Playwright <a href="https://playwright.dev/docs/intro"><img src="https://playwright.dev/img/playwright-logo.svg" alt="Node.js" height="22" /></a>  and CodeceptJS <a href="https://codecept.io/"><img src="https://codecept.io/logo.svg" alt="Node.js" height="22" /></a> with BrowserStack <a href="https://browserstack.com"><img src="https://www.browserstack.com/blog/favicon.png" alt="BrowserStack" height="22" /></a>
+
+
+## Setup
+*  Clone the repo `git clone -b sdk https://github.com/browserstack/codecept-js-playwright-browserstack.git` and run `cd codecept-js-playwright-browserstack`.
```
Comment (Baaryan) [inline README.md]: Let's move these to separate lines

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+  export BROWSERSTACK_USERNAME=<browserstack-username> &&
+  export BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+  ```
+* You can view your test results on the [BrowserStack Automate dashboard](https://www.browserstack.com/automate)
+* Understand how many parallel sessions you need by using our [Parallel Test Calculator](https://www.browserstack.com/automate/parallel-calculator?ref=github)
+
+## Addtional Resources
+* [Documentation for writing Automate test scripts in CodeceptJS](https://codecept.io/helpers/WebDriver/#webdriver)
+* [Documentation for playwright](https://playwright.dev/docs/intro)
+* [Customizing your tests on BrowserStack](https://www.browserstack.com/automate/capabilities)
```
Comment (Baaryan) [inline README.md:30]: Please sync once with Product for readme review and links

**Review by Baaryan — COMMENTED** [review]
Overall looks good to me

File: README.md (hunk 1 above)
Comment (shayan-bstack) [reply README.md]: https://github.com/browserstack/codecept-js-playwright-browserstack/pull/1/commits/835649808b4667c4fb31a9a69bc1f9af62cfa084

**Review by RutvikChandla — APPROVED** [review]

**Review by pranavj1001 — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,32 @@
-# codecept-js-playwright-browserstack
-Sample code for running sessions with Codecept Playwright on Browserstack
+# CodeceptJS-Playwright-Browserstack
+
+# Integrating Playwright <a href="https://playwright.dev/docs/intro"><img src="https://playwright.dev/img/playwright-logo.svg" alt="Node.js" height="22" /></a>  and CodeceptJS <a href="https://codecept.io/"><img src="https://codecept.io/logo.svg" alt="Node.js" height="22" /></a> with BrowserStack <a href="https://browserstack.com"><img src="https://raw.githubusercontent.com/browserstack/MSTest-playwright-browserstack/main/assets/browserstack.png" alt="BrowserStack" height="22" /></a>
+
+
+## Setup
+*  Clone the repo `git clone -b sdk https://github.com/browserstack/codecept-js-playwright-browserstack.git`.
```
Comment (pranavj1001) [inline README.md]: Let's confirm the branch `main` or `sdk`. Rest lgtm

Comment (shayan-bstack) [reply README.md]: It should be main ( got this info from PM) , changed the readme accordingly

File: README.md [hunk 4]
```diff
@@ -1,2 +1,32 @@
-# codecept-js-playwright-browserstack
-Sample code for running sessions with Codecept Playwright on Browserstack
+# CodeceptJS-Playwright-Browserstack
+
+# Integrating Playwright <a href="https://playwright.dev/docs/intro"><img src="https://playwright.dev/img/playwright-logo.svg" alt="Node.js" height="22" /></a>  and CodeceptJS <a href="https://codecept.io/"><img src="https://codecept.io/logo.svg" alt="Node.js" height="22" /></a> with BrowserStack <a href="https://browserstack.com"><img src="https://raw.githubusercontent.com/browserstack/MSTest-playwright-browserstack/main/assets/browserstack.png" alt="BrowserStack" height="22" /></a>
+
+
+## Setup
+*  Clone the repo `git clone -b main https://github.com/browserstack/codecept-js-playwright-browserstack.git`.
```
Comment (pranavj1001) [inline README.md]:
```suggestion
*  Clone the repo `git clone https://github.com/browserstack/codecept-js-playwright-browserstack.git`.
```
`main` is the default branch. Can remove the branch arg.

**Review by pranavj1001 — APPROVED** [review]

**Pull request merged**